src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -1,8 +1,28 @@
+// Node modules.
 import React from 'react';
 import moment from 'moment';
-
+// Relative imports.
 import * as customPropTypes from '../prop-types';
 
+const onDownloadLink = (form) => (event) => {
+  // Derive if the download is a PDF.
+  const isPDF = form?.attributes?.url.toLowerCase().includes('.pdf');
+
+  // IE10+ : (has Blob, but not a[download] or URL)
+  if (navigator.msSaveBlob && isPDF) {
+    event.preventDefault();
+
+    // Make the request to download the PDF.
+    const xhr = new XMLHttpRequest();
+    xhr.open('GET', form?.attributes?.url, true);
+    xhr.responseType = 'blob';
+    xhr.onload = (event) => {
+      const pdf = event.target.response;
+      navigator.msSaveBlob(pdf, `${form?.id}.pdf`);
+    };
```

Comment:
I'm guessing no, but to be sure - `fetch` can't be used for this?

Reply:
I wish haha but had to use this due to contentType = 'blob'